<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>436</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="customUI">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtNameStr">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
padding: 4px;
border-style: solid;
border: 1px solid gray;
border-radius: 10px;
font:  bold 11px
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Prefix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtNamePrefix">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
padding: 3px;
border-style: solid;
border: 1px solid gray;
border-radius: 10px;
font:  bold 11px
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Suffix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtNameSuffix">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
padding: 4px;
border-style: solid;
border: 1px solid gray;
border-radius: 10px;
font:  bold 11px
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Select By Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtSelectByName">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
padding: 4px;
border-style: solid;
border: 1px solid gray;
border-radius: 10px;
font:  bold 11px
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ldtFind">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
padding: 4px;
border-style: solid;
border: 1px solid gray;
border-radius: 10px;
font:  bold 11px
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Replace By:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtReplaceStr">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
padding: 4px;
border-style: solid;
border: 1px solid gray;
border-radius: 10px;
font:  bold 11px
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkHierrachy">
       <property name="toolTip">
        <string>apply to whole Hierrachy</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnUpperCase">
       <property name="text">
        <string>UpperCase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLowerCase">
       <property name="text">
        <string>LowerCase</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
